### PR TITLE
ramiray/c01-git01

### PR DESCRIPTION
**Pull request by ramiray**
# Description

Exercise (e.g. C01-E01): 

Student Name:

> A brief explanation of what has been done.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-git01/ramiray/Readme.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# c01-git01 - Below commands are used for this exercise . 
+
+1. Merging Exercise:
+```
+git checkout master
```
Comment (christina-roperto) [inline classes/01class/exercises/c01-git01/ramiray/Readme.md:5]: Hi Rami, not sure if you've done a different exercise, the first one is ``level intro3`` which is the third exercise of introduction sequence. Below is the instructions from their website

```
To complete this level, do the following steps:
- Make a new branch called bugFix
- Checkout the bugFix branch with git checkout bugFix
- Commit once
- Go back to master with git checkout
- Commit another time
- Merge the branch bugFix into master with git merge
```

File: classes/01class/exercises/c01-aws02/ramiray/Readme.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+1. Created s3 bucket using following command. 
+
+```
+aws s3 mb s3://ramyadevopstest --region ap-southeast-2
+make_bucket: s3://ramyadevopstest
+
+```
```
Comment (christina-roperto) [inline classes/01class/exercises/c01-aws02/ramiray/Readme.md:7]: Hey Rami,
this one is from another exercise. Did you forget to checkout from master before submitting the new one? Happens to most of us! 😄  Let us know if you need any help fixing this issue

**Comment by ramiray** [comment]
Hi Team,

Sure I will make the changes and resubmit them . Currently I'm super busy
with shifting my house and need some time to catch up with these changes
and new exercises .

Thanks ,
Ramya

File: classes/01class/exercises/c01-aws02/ramiray/Readme.md (hunk 2 above)
Comment (ramiray) [reply classes/01class/exercises/c01-aws02/ramiray/Readme.md:7]: Hi Christina,

I have updated the c01-aws02 file. 

Thanks,
Ramya

File: classes/01class/exercises/c01-git01/ramiray/Readme.md (hunk 1 above)
Comment (roperto) [reply classes/01class/exercises/c01-git01/ramiray/Readme.md:8]: nice, marking as resolved

**Comment by roperto** [comment]
Hi @ramiray -- it seems the changes that @christina-roperto requested were fixed.

It seems that you accidentally added the answers of c01-aws02 in this pull request too. The c01-git01 is correct, but the other one needs to be removed (one PR per exercise).

It may seem a bit confusing if you are new to git, but it is not too hard to remove it.

Feel free to reach out via slack to #public-academy-jun-2020 -- christina or I will be happy to help -- should only take a few minutes

Cheers

**Comment by ramiray** [comment]
Hi Daniel,

I have made all the necessary changes for all the Class01 exercises. I
understood the git PR much better now :)

Thanks ,
Ramya

**Pull request merged**